src/lib/messenger/passback-refresh.ts

```diff
@@ -0,0 +1,30 @@
+import { isString } from '@guardian/libs';
+import type { RegisterListener } from 'core/messenger';
+import { getAdvertById } from '../dfp/get-advert-by-id';
+import { refreshAdvert } from '../dfp/load-advert';
+
+const passbackRefresh = (specs: string, adSlot: HTMLElement) => {
+	const advert = getAdvertById(adSlot.id);
+	if (advert) {
+		advert.slot.setTargeting('passback', specs);
+		refreshAdvert(advert);
+	}
+};
+
+const init = (register: RegisterListener): void => {
+	register('passback-refresh', (specs, ret, iframe) => {
```

Comment:
What is ret being used for? Could we add a comment to explain why it's declared but not used?

Reply:
It's not used but we needed access to the third argument, will change to `_`